web/features/site-metadata/SiteMetadata.tsx

```diff
@@ -8,14 +8,25 @@ type SiteMetadataProps = {
   image?: string;
   author?: string;
   canonicalUrl?: string;
+  contentType?: string;
```

Comment:
Could this be a union of allowed strings, that implement the open graph standard?